backend/cdb/api/db/crud/deployment.py

```diff
@@ -0,0 +1,31 @@
+from uuid import UUID
+
+from asyncpg import Record
+from asyncpg.connection import Connection
+
+from cdb.api.db.models.deployment import Deployment, DeploymentInput
+
+
+async def insert_deployment(connection: Connection, deployment_info: DeploymentInput):
+    result: Record = await connection.fetchrow(
```

Comment:
```suggestion
    result: Record | None = await connection.fetchrow(
```
Si tu fais un `if result` plus bas c'est qu'il peut être nul non ?